subsys/net/openthread/Kconfig

```diff
@@ -52,6 +52,12 @@ config OPENTHREAD_NRF_SECURITY
 config OPENTHREAD_MBEDTLS_LIB_NAME
 	default "mbedtls_external" if OPENTHREAD_NRF_SECURITY
 
+config OPENTHREAD_MBEDTLS_DEBUG
+	bool "MbedTLS logs for OpenThread"
+	select MBEDTLS
```

Comment:
Nit: Maybe a bit much for MBEDTLS_DEBUG to enable the MBEDTLS feature itself.

This is already done by OPENTHREAD_MBEDTLS, I'm thinking that should be enough.

Reply:
The problem is we use OPENTHREAD_NRF_SECURITY (not OPENTHREAD_MBEDTLS). MBEDTLS is not selected in our default configuration.